Formula/glibmm.rb

```diff
@@ -18,20 +18,16 @@ class Glibmm < Formula
 
   depends_on "meson" => :build
   depends_on "ninja" => :build
-  depends_on "pkg-config" => :build
+  depends_on "pkg-config" => [:build, :test]
   depends_on "glib"
   depends_on "libsigc++"
 
   fails_with gcc: "5"
 
   def install
-    ENV.cxx11
```

Comment:
Should be C++17. I think Meson script handles this https://gitlab.gnome.org/GNOME/glibmm/-/blob/2.74.0/meson.build#L7